src/app/interceptors/http/bad-request.interceptor.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+
```

Comment:
Remove this line

Reply:
why? I don't understand this. I use them during the code

Reply:
line 2